source/includes/cli-guide/_install.html.md

```diff
@@ -149,45 +159,50 @@ cd gym-mountaincar-sample
 pip3 install -r requirements.txt
 ```
 
-Enter into the Mountain Car folder and then install the Mountain Car requirements from OpenAI Gym.
+Enter into the Mountain Car folder and then install the Mountain Car requirements from OpenAI Gym with pip.
 
-# Create your BRAIN
 
-Create your BRAIN and give it a name. You can also create a BRAIN from your [beta.bons.ai][5] BRAIN Dashboard instead of using the command line. Either way, you will view your BRAIN's progress on the BRAIN Details page.
 
-### Create from Web
 
-![BRAINS Dashboard Page][15]
+# Create Your BRAIN
 
-The dashboard has a NEW BRAIN button, pictured above. Click on it to start the BRAIN creation process, where you're provided a form to give your brain a name and description. Name can contain alpha, numerals, or hyphens; the description is arbitrary and should contain human-readable text that explains your intent for the BRAIN.
+```
+bonsai create myMountainCarBrain
+```
 
-### Create from Command Line
+Create your BRAIN (and its related project file) with [`bonsai create`][18] and give it a name. We used *myMountainCarBrain* in the example code but you can name it whatever you'd like. You will view your BRAIN's training progress on the BRAIN's Details page which can be found on the [BRAIN dashboard][5].
 
 ```
-bonsai create myMountainCarBrain
+bonsai push
 ```
-You may give your brain a name with `bonsai create brainName` but there is no option for a description with this method but after you create your BRAIN, you can change its description from the web on the BRAIN Details page.
 
-# Train your BRAIN
+Uae [`bonsai push`][19] to upload the cloned Mountain Car project and its associated files to the Bonsai AI Engine for training.
+
+
+
+
+# Train Your BRAIN
+
+Almost there! Time to tell the Bonsai AI Engine to prepare a new BRAIN version for training. There are currently two options for training a BRAIN, one is to run a simulator remotely using Bonsai's servers, and the other is to run a simulator locally with your own computer.
+
+Most common simulators can be trained remotely on our servers such as EnergyPlus, OpenAI Gym's, and simple simulators all found on our [Examples documentation][23]. If the simulator is not one that we support remotely, you'll need to run it locally.
```

Comment:
I'd leave off `most common` - `Simulators which can be trained remotely include EnergyPlus ...`